src/typescript/frontend/src/pages/launch-emojicoin/index.tsx

```diff
@@ -14,10 +14,10 @@ const LaunchEmojicoinPage: React.FC = () => {
   const { t } = useTranslation();
   const { theme } = useThemeContext();
 
-  const [emoji, setEmoji] = useState<EmojiClickData | null>(null);
+  const input = document.querySelector("input");
```

Comment:
```suggestion
  const input = inputRef.current;
```

and later:

```typescript
  // later, <Input ... ref={inputRef} />
```

Can we convert this to a ref with the input element, ideally we'd adhere to React best practices, but let me know if perhaps there's more going on here that I'm not understanding